docs_outdoornav_user_manual/faq.mdx

```diff
@@ -104,7 +104,7 @@ ping 192.168.131.30
     Enable the visualization of the [path contraint](./web_user_interface/ui_autonomous_mode.mdx#constrained_path) and Teleop the robot back into the
     navigable space.
 
-6.  **How can I remove certain sensors from the collision avoidance pipeline?**
+6.  **How can I remove certain sensors from the collision avoidance!/ pipeline?**

```

Comment:
Was this change deliberate? It looks like a typo on first glance but I could be wrong.